emhttp/plugins/dynamix/include/.login.php

```diff
@@ -81,24 +81,64 @@ function cleanupFails(string $failFile, int $time): int
 
     // Save fails to disk
     if ($updateFails) {
-        $failText = implode("\n", $fails)."\n";
+        $failText = implode("\n", $fails) . "\n";
         writeToFile($failFile, $failText);
     }
     return count($fails);
 }
 
+
 function verifyUsernamePassword(string $username, string $password): bool
 {
-    if ($username != "root") {
-        return false;
-    }
+    if ($username != "root") return false;
```

Comment:
Please add a comment explaining what values the "login as unraid.net" button submits to this function, it is not immediately obvious how SSO gets past this `if` statement or the `strlen($password)` check below